### PR TITLE
fix #1 make value private, add getValue()

### LINKED ISSUE #1: Support getting the current value
Need to implement an API to return the current value of the storage. 

## Suggestion

```js
const store = new LiveStorage(...)

store.getValue() // or
store.current() // 
```

### PR DESCRIPTION
**Pull request by Aprillion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +43,9 @@ const posts = new LiveStorage('posts', [])
 // from the previous storage value.
 posts.update((prevPosts) => prevPosts.concat({ title: 'Brave new world' });
 ```
+
+### Get value
+
+```js
+console.log(posts.getValue())
```
Comment (kettanaito) [inline README.md]: I'd remove the console call and added a comment showing what would be the value.

```js
posts.getValue() // [{ title: '...' }, { title: '...' }]
```

**Comment by kettanaito** [comment]
Could you please double check your Git configuration for this repo? It looks like the email you provision commits doesn't match your GitHub user's email.

**Comment by Aprillion** [comment]
> Could you please double check your Git configuration for this repo? It looks like the email you provision commits doesn't match your GitHub user's email.

uh, I need to look for some script to set correct email for work vs github repos without manual config

**Comment by Aprillion** [comment]
email in the commit fixed, README updated

**Review by kettanaito — APPROVED** [review]
Love this. Awesome addition, @Aprillion!

**Pull request merged**